### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by dxvladislavvolkov**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+name: Bug report
+about: Report a bug in devextreme-angular
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- *************************************************************************************************
+To help us expedite the issue processing, please search opened and closed tickets in our support center (https://www.devexpress.com/Support/Center/Question/List/1) before you submit a new issue. The tickets can contain resolutions, workarounds, or information about the progress in fixing.
```
Comment (dirk-pieterse) [inline .github/ISSUE_TEMPLATE/bug_report.md]: the issue processing?

Comment (dirk-pieterse) [reply .github/ISSUE_TEMPLATE/bug_report.md]: the progress in fixing?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+<!-- Please provide the following information -->
+
+**Package versions:**
+
+    devexteme version:
+    devextreme-angular version:
+
+**Steps to reproduce:**
+<!-- Provide a sample application that illustrates the bug if possible. Do it using:
```
Comment (dirk-pieterse) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Do it using?

Comment (dirk-pieterse) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Do not use 'using'

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (TikhonovaTatyana) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
To reduce the time it takes to process issues, search opened and closed tickets in our support center (https://www.devexpress.com/Support/Center/Question/List/1) before you submit a new issue. The tickets can contain resolutions, workarounds, or information about fixes.
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (TikhonovaTatyana) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
<!-- Provide a sample application that illustrates the bug if possible. You can use the following services: 
```

**Pull request merged**